### PR TITLE
Fix formatting of OpenID Connect examples

### PR DESCRIPTION
**Pull request by afairb**
### Why:

There are a few small formatting issues on this page of documentation.

Closes: n/a (small copy fix only)

### What's being changed (if available, include any code snippets, screenshots, or gifs):

1. One code sample is missing an opening ` symbol
2. Other code samples have mismatched `"` symbols

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by sineels — COMMENTED** [review]
#의 중복

**Comment by cmwilson21** [comment]
@afairb Thanks so much for submitting a PR! I'll get this triaged for review :zap:

**Review by subatoi — DISMISSED** [review]
Thank you very much for making this series of changes 👍 ✨

**Comment by subatoi** [comment]
@afairb Apologies, I just need to confirm internally we can make an exception on this one before merging, as a matter of course, as it's one of the articles that normally requires approval from another team.

**Comment by afairb** [comment]
> @afairb Apologies, I just need to confirm internally we can make an exception on this one before merging, as a matter of course, as it's one of the articles that normally requires approval from another team.

No problem - I didn't realise these files were in that list before raising the PR and seeing the bot comment!

File: content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md
Comment (N-Usha) [inline content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md:293]: What is the change suggested here?

**Review by N-Usha — APPROVED** [review]

File: content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md
Comment (subatoi) [reply content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md:293]: @N-Usha it's mainly just formatting around some of the monospaced element:

> `"job_workflow_ref:octo-org/octo-automation/.github/workflows/oidc.yml@refs/heads/main"`

It's got a little bit broken in the diff because of a newline, I think, which has obscured the relevant change in the diff.

@afairb 👋 would you mind editing this slightly? It won't matter for the render in the site but it would be ideal if we could do. Then we can get this merged!

Comment (afairb) [reply content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md]: Sorry, I hadn't spotted this earlier. Crept in by accident and removed now :)

**Review by subatoi — APPROVED** [review]
Thank you again for fixing this and for your interest in the docs! I'll go ahead and get this merged

**Pull request merged**
